server/db/postgres/sequelize_config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  development: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_development',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  test: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_test',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  production: {
+    use_env_variable: 'POSTGRES_DB_URL'
+  }
```

Comment:
Do we need to specify the dialect as well as other config here?

Otherwise we could add some docs on how to test production locally on your machine